stable/admin/templates/role.yaml

```diff
@@ -34,11 +34,13 @@ rules:
   - create
   - update
 {{- if eq (include "defaulttrue" .Values.nuodb.addClusterRoleBinding) "true" }}
+{{- $namespace := default .Release.Namespace .Values.admin.namespace -}}
+{{- $release := .Release.Name }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: nuodb-kube-inspector
+  name: {{ $namespace }}-{{ $release }}-nuodb-kube-inspector
```

Comment:
Use `{{ include "admin.fullname" . }}` instead of the release name directly.

Reply:
I understand the value of using `nameOverride` or `fullNameOverride` instead of release name, but why also include the `domain` and `cluster`? It seems to be creating a very long and very complicated name for the resource when all we really want is to reduce the risk of name collisions.

Reply:
This is the name that is used as a prefix in every resource associated with the release. All the ConfigMaps have it as a prefix, it is used as the StatefulSet name, etc. Why use something different?